demo/src/App.js

```diff
@@ -1,80 +1,50 @@
-import React, { PureComponent } from 'react';
-import Slider from '@material-ui/core/Slider';
-import { withStyles } from '@material-ui/core/styles';
-import { MicroscopyViewer } from '../../src';
-import { source } from './source-info';
-import './App.css';
-
-const RedSlider = withStyles({
-  root: {
-    color: 'red'
-  }
-})(Slider);
-const BlueSlider = withStyles({
-  root: {
-    color: 'blue'
-  }
-})(Slider);
-const GreenSlider = withStyles({
-  root: {
-    color: 'green'
-  }
-})(Slider);
-const OrangeSlider = withStyles({
-  root: {
-    color: 'orange'
-  }
-})(Slider);
+import React, { PureComponent } from "react";
+import Slider from "@material-ui/core/Slider";
+import { withStyles } from "@material-ui/core/styles";
+import { MicroscopyViewer } from "../../src";
+import { source } from "./source-info";
+import "./App.css";
 
 export default class App extends PureComponent {
   constructor(props) {
     super(props);
-    this.handleRedSliderChange = this.handleRedSliderChange.bind(this);
-    this.handleGreenSliderChange = this.handleGreenSliderChange.bind(this);
-    this.handleBlueSliderChange = this.handleBlueSliderChange.bind(this);
-    this.handleOrangeSliderChange = this.handleOrangeSliderChange.bind(this);
     this.resize = this.resize.bind(this);
+    var sliderValues = {};
```

Comment:
One of the eslint warnings is var: Let's find a way to use const instead.

Reply:
Is `let` ok?

Reply:
nevermind i forgot you can set a const object.